ecs_deploy/cli.py

```diff
@@ -327,7 +331,7 @@ def run(cluster, task, count, command, env, secret, launchtype, subnet, security
 
         td = action.get_task_definition(task)
         td.set_commands(**{key: value for (key, value) in command})
-        td.set_environment(env)
+        td.set_environment(env, env_file)
```

Comment:
Same problem as above. Please also consider copying the `--exclusive-env` option from another operation to this operation, too